ext/RMagick/rmagick.h

```diff
@@ -46,6 +46,13 @@
 
 #include "extconf.h"
 
+#ifndef TRUE
```

Comment:
Can we do `#define` here instead of `#ifndef`, or are there cases when it might already be defined?

Reply:
Since `TRUE` is a common name, it might be add in other header files.
So, i think it is better to guard.

Without definition, it caused compile error....